### PR TITLE
feat: add a few stores for spain

### PR DESCRIPTION
**Pull request by PaoloOranges**
### Description
- New Stores from Spain:
  - ldlc ES
  - Asus ES
  - eletronicamente
  - opirata
  - ultimainformatica
  - siabyte
  - game ES
  - neobyte
  - extremmedia
 
- Added 3080ti and 3070ti to NVidia ES
- Added 3080ti to PCComponentes.
- Added 3090 and 3080ti to coolmod
- Added 3080ti to vsgamers
- Update a few 3080/3090 to Amazon ES and vsgamers

No new fixes

### Testing

All stores and models added are running live in a local machine.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -439,15 +440,15 @@ const store = {
   showOnlySeries: envOrArray(process.env.SHOW_ONLY_SERIES, [
     '3060',
     '3060ti',
-    '3070',
-    '3080',
+    // '3070',
+    '3070ti',
```
Comment (jef) [inline src/config.ts:444]:
```suggestion
    '3070',
    '3070ti',
```

File: src/config.ts [hunk 2]
```diff
@@ -439,15 +440,15 @@ const store = {
...
     '3080ti',
     '3090',
     'rx6700xt',
     'rx6800',
     'rx6800xt',
     'rx6900xt',
-    'ryzen5600',
+    //'test:series'
```
Comment (jef) [inline src/config.ts:451]:
```suggestion
    'test:series'
```

**Review by jef — COMMENTED** [review]
Overall looks great! Do you mind uncommenting those other series and adding the stores to the documentation?

Thanks!

File: src/config.ts (hunk 2 above)
Comment (PaoloOranges) [reply src/config.ts]: Hi Jef, thanks for the feedback. I've fixed what you said and I hope to have updated correctly the doc, I was not sure what to edit :)

**Review by jef — APPROVED** [review]
Thank you!

**Pull request merged**